src/components/book.css

```diff
@@ -1,9 +1,9 @@
-@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto+Slab&display=swap');
 
 .card {
   margin-left: 1.5rem;
   margin-right: 1.5rem;
-  font-family: RobotoSlab, serif;
+  font-family: "Montserrat", serif;
 }
 
 .cardlist {
```

Comment:
- [ ] Please🙏🙏 take into consideration the alignment of the navbar logo and user Circle icon with the rest of the elements on the page.
- [ ]  To achieve this, it's recommended to add left and right margins or padding. 
- [ ] Furthermore, in order to adhere to the design, consider reducing the vertical spacing between the cards.
- [ ] By implementing these adjustments, your design will not only align more closely with the project requirements but also stand out effectively.🎉🥱